### ISSUE TITLE
[BUG] Server is not able to load Skins

### ISSUE THREAD

**Issue by Bleachhub** [Armourers-Workshop/Armourers-Workshop#361]
**Describe the bug**
When any loaded Skin-Blocks are being placed, they either vanish or are invisible.

**To Reproduce**
1. Go to 'the Global Skin Galery'
2. Click on 'Select a Block-Skin'
3. Load the Skin into your 'Inventory'
4. Place the Skin
5. Skin vanish or is invisible (Errors in the Console)

**Expected behavior**
Skin is being placed and showing up for all players.

**server log**
[08-05-2021 17:51:10] Downloaded skin Skin [properties=SkinProperties [properties={authorName=VermillionX, customName=HD Flatscreen}], type=BLOCK] from client EntityPlayerMP['Name'/236, l='bleach-soul-blades', x=248.52, y=28.00, z=-464.92]
[08-05-2021 17:51:13] Block skin at x:247 y:28 z:-472 has no tile entity.
[08-05-2021 17:51:13] Block skin at x:247 y:28 z:-472 has no skin data.
[08-05-2021 17:51:13] Block skin at x:247 y:28 z:-472 has no tile entity.
[08-05-2021 17:51:16] Block skin at x:247 y:28 z:-474 has no tile entity.
[08-05-2021 17:51:16] Block skin at x:247 y:28 z:-474 has no skin data.
[08-05-2021 17:51:16] Block skin at x:247 y:28 z:-474 has no tile entity.
[08-05-2021 17:51:16] Block skin at x:247 y:28 z:-472 has no tile entity.
[08-05-2021 17:51:16] Block skin at x:247 y:28 z:-472 has no skin data.
[08-05-2021 17:51:16] Block skin at x:247 y:28 z:-472 has no tile entity.
[08-05-2021 17:51:16] Block skin at x:247 y:28 z:-470 has no tile entity.
[08-05-2021 17:51:16] Block skin at x:247 y:28 z:-470 has no skin data.
[08-05-2021 17:51:16] Block skin at x:247 y:28 z:-470 has no tile entity.

**Additional context**
I've just updated the Server from 1.7.10 to 1.12.2 and already deleted all data from the skin-libary (got a backup if necessary), but either with or without data in the folder nothing is loading, reinstalling the mod also didn't worked and I'm not sure if there's something missing on the server or client.

**Comment by RyuGituwu** [comment]
Similar Issue here (As well with having the Bleach Mod - At least I suppose you do, correct me if I'm wrong).
The server has been updated from 1.7 to 1.12 and I've checked with our Team which told us that the previous Map (Beta) is too old and it's not working because of that.

After changing the map it worked and we've now spent building for about 2-3 months on a new map, but now no custom blocks (Only Blocks) are showing up :/
I'll paste the Logs below and attach a Screenshot.

> [16-11-2021 23:01:15] Loaded file form lib: LibraryFile [fileName=Barrel, filePath=/official/, skinType=moe.plushie.armourers_workshop.common.skin.type.block.SkinBlock@19753714, directory=false]
[16-11-2021 23:01:57] Block skin at x:0 y:199 z:-2 has no tile entity.
[16-11-2021 23:01:57] Block skin at x:0 y:199 z:-2 has no skin data.
[16-11-2021 23:01:57] Block skin at x:0 y:199 z:-2 has no tile entity. 
![uwu](https://user-images.githubusercontent.com/94488897/142065856-e92e8d48-0c11-4a83-8748-0fa14863dce4.PNG)

Haven't found anything similar besides that thread here, so yee... Might try some troubleshooting on the weekend and will follow up, but feel free to reply if anyone knows how to resolve this.
